src/openapi/aptos/aptos.yaml

```diff
@@ -1127,6 +1205,78 @@ paths:
                   summary: Example string response
                   value: "0x81e2e2499407693c81fe65c86405ca70df529438339d9da7a6fc2520142b591e"
 
+  /v1/tables/{table_handle}/raw_item:
+    post:
+      summary: "/v1/tables/{table_handle}/raw_item"
+      description: >
+        Get a raw table item at a specific ledger version from the table identified by `{table_handle}` and the key provided in the request body.
+
+        The `get_raw_table_item` endpoint only requires a serialized key (hex string) and returns the value in binary canonical serialization (BCS) format.
+
+        The Aptos nodes prune account state history via a configurable time window. If the requested ledger version has been pruned, the server responds with a 410.
+      tags:
+        - Aptos API Endpoints
+      security:
+        - apiKey: []
+      parameters:
+        - name: table_handle
+          in: path
+          required: true
+          description: >
+            Table handle — a hex encoded 32-byte Aptos address (with or without 0x prefix).
+          schema:
+            type: string
+            format: hex
+          example: "0x982be82410ddc6480dc8976e2866a2c7d162ee9d02d97ee842ac38b0b105086"
+        - name: ledger_version
+          in: query
+          required: false
+          description: >
+            Ledger version to query state from. If omitted, the latest version will be used.
+          schema:
+            type: string
+            format: uint64
+          example: "1830800010"
+        - name: Accept
+          in: header
+          required: true
+          description: >
+            Must be set to `application/x-bcs` to receive the response in raw BCS format.
+          schema:
+            type: string
+            enum: ["application/x-bcs"]
+          example: "application/x-bcs"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - key
+              properties:
+                key:
+                  type: string
+                  format: hex
+                  description: >
+                    Serialized BCS key for the item, as a hex-encoded string prefixed with `0x`.
+            example:
+              key: "0x9b363c90098d441b4922d547a28613590d3c7cc99c0bb6e78c1695fd34d977a2"
```

Comment:
just a heads up that using this example in the request explorer results in a 404, not sure if we have an example that would work that we can replace it with! https://alchemy-preview-881f2286-6071-401f-8639-2bb2e11a7933.docs.buildwithfern.com/docs/node/aptos/aptos-api-endpoints/aptos-api-endpoints/v-1-tables-table-handle-raw-item?explorer=true